templates/crud/controller/Controller.tpl.php

```diff
@@ -45,7 +45,7 @@ public function new(Request $request, EntityManagerInterface $entityManager): Re
 
         return $this->render('<?= $templates_path ?>/new.html.twig', [
             '<?= $entity_twig_var_singular ?>' => $<?= $entity_var_singular ?>,
-            'form' => $form,
+            'form' => $form->createView(),
```

Comment:
`render()` will automatically call `createView()` on all parameters that are an instance of `FormInterface`!

Reply:
I know that it does... If not yet done. The question is: shouldn't this be done?

Pros: more explicit.
Cons: doesn't handle the 422 code, for instance (like Turbo doesn't like this).

I'm split on that. Hence the questioning! :)